### ISSUE TITLE
tabbing through python walkthrough doesn't expand options

### ISSUE THREAD

**Issue by eleanorjboyd** [microsoft/vscode#229588]
Testing #229460

here when I go through using tab, so pressing tab, tab, tab etc it will not open any submenus (I would instead how to expand them by pressing enter). On the other hand when I use the arrow keys, up and down, it does it expand the submenus but keeps my selection on the entire submenu section. These two feel like they should have consistent behavior unless that is intended and there is another way to do the reverse of tab that I should be using in this instance insteadhttps://github.com/user-attachments/assets/c4c55bed-c9c2-4ee1-9518-4ea5db48a905

**Comment by bhavyaus** [comment]
Cleaned up tabbing behavior. Keeping behavior similar to some other UX.
Tab is to navigate between the steps. Enter to select a step and then tab to into the elements with in a step.

**Comment by aiday-mar** [comment]
Hi I am not sure what the expected behavior now is. Could I ask for some verification steps?
